modules/boards/base.nix

```diff
@@ -73,33 +73,5 @@
         "input_leds"
       ];
     };
-
-    hardware = {
-      # driver & firmware for Mali-G610 GPU
-      # it works on all rk2588/rk3588s based SBCs.
-      opengl.package =
-        (
-          (pkgs.mesa.override {
-            galliumDrivers = ["panfrost" "swrast"];
-            vulkanDrivers = ["swrast"];
-          })
-          .overrideAttrs (_: {
-            pname = "mesa-panfork";
-            version = "23.0.0-panfork";
-            src = pkgs.fetchFromGitLab {
-              owner = "panfork";
-              repo = "mesa";
-              rev = "120202c675749c5ef81ae4c8cdc30019b4de08f4"; # branch: csf
-              hash = "sha256-4eZHMiYS+sRDHNBtLZTA8ELZnLns7yT3USU5YQswxQ0=";
-            };
-          })
-        )
-        .drivers;
-
-      enableRedistributableFirmware = lib.mkForce true;
-      firmware = lib.mkIf config.hardware.mali.enableFirmware [
```

Comment:
Don't delete that - it is my change from https://github.com/ryan4yin/nixos-rk3588/pull/35 and this code is used for OpenCL, which does work without issues. And even if you delete that, you should also delete corresponding options, breaking backwards compatibility explicitly.